### PR TITLE
add basic structure and explanation for fake api

### PR DESCRIPTION
**Pull request by tynx**
each subfolder should/will be populated with version files...

this is a basic draft based on #41

### CODE REVIEW

**Comment by stripedpajamas** [comment]
I am thinking that we will only support the `stable` release, as at least currently the whole build process works by looking at the latest tag on the MS repo. So I would say remove the insider folders from this PR. 

Also, I don't think we will need to host the repo on Github pages. It should be that if the file is there (and updated at build time), then VSCodium could access it directly at https://github.com/vscodium/vscodium/master/api/update/{os}/stable/VERSION. So that part can be removed from the readme.

File: api/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Faking the update-API of Microsoft
+
+There is a distinction between stable and insider for the "quality" of the version. Also there are three platforms: linux, win32 and darwin. For each version there is a subfolder. There should be a file in each of the folders called "VERSION" (uppercase).
+
+By example of the api of MS itself (darwin/stable):
+https://vscode-update.azurewebsites.net/api/update/darwin/stable/VERSION
+
+The repo containing the version-json-files should have pages activated, see project settings. Instructions here: https://pages.github.com/
```
Comment (stripedpajamas) [inline api/README.md]: This line can be removed (shouldn't need to use Github pages)
```suggestion
```

File: api/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Faking the update-API of Microsoft
+
+There is a distinction between stable and insider for the "quality" of the version. Also there are three platforms: linux, win32 and darwin. For each version there is a subfolder. There should be a file in each of the folders called "VERSION" (uppercase).
```
Comment (stripedpajamas) [inline api/README.md]:
```suggestion
There is a distinction between stable and insider for the "quality" of the version. Also there are three platforms: linux, win32 and darwin. For each platform there is a subfolder for the quality. There should be a file in each of the folders called "VERSION" (uppercase).
```

**Comment by JL2210** [comment]
@stripedpajamas You can commit directly to his repository if he's not responding.

**Comment by stripedpajamas** [comment]
PR seems to be abandoned and more work needs to go into signing the build output before any updating API can be considered.

**Pull request closed without merging**